backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/MigrateVmCommand.java

```diff
@@ -423,6 +427,10 @@ private MigrateVDSCommandParameters createMigrateVDSCommandParameters() {
         if (parallelMigrations == null) {
             maxIncomingMigrations = maxOutgoingMigrations = effectiveMigrationPolicy.getMaxMigrations();
         }
+        if (getVm().getCpuPinningPolicy() == CpuPinningPolicy.DEDICATED) {
+            cpuSets = CpuPinningHelper.getAllPinnedPCpus(getVm().getCurrentCpuPinning()).stream()
+                    .map(Object::toString).collect(Collectors.toSet());
```

Comment:
don't we have to have that collection sorted for VDSM?

Reply:
yes, but i remember it was sorted here but no harm in adding here as well.